tools/report-converter/codechecker_report_converter/cppcheck/analyzer_result.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import glob
+import logging
+import os
+import plistlib
+import sys
+
+from codechecker_report_converter.analyzer_result import AnalyzerResult
+from codechecker_report_converter.report import generate_report_hash
+
+
+LOG = logging.getLogger('ReportConverter')
+
+
+class CppcheckAnalyzerResult(AnalyzerResult):
+    """ Transform analyzer result of Cppcheck. """
+
+    TOOL_NAME = 'cppcheck'
+
+    def parse(self, analyzer_result):
+        """ Creates plist objects from the given analyzer result.
+
+        Returns a list of plist objects.
+        """
+        plist_files = []
+        if os.path.isdir(analyzer_result):
+            plist_files = glob.glob(os.path.join(analyzer_result, "*.plist"))
+        elif os.path.isfile(analyzer_result) and \
+                analyzer_result.endswith(".plist"):
+            plist_files = [analyzer_result]
+        else:
+            LOG.error("The given input should be an existing CppCheck result "
+                      "directory or a plist file.")
+            sys.exit(1)
```

Comment:
the same thinking here what I mentioned about the sys.exit, wouldn't be better to do it only at the main script?